packages/cardano-services/src/Projection/migrations/1726479765002-nft-metadata-index.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { NftMetadataEntity } from '@cardano-sdk/projection-typeorm';
+
+export class NftMetadataIndexMigration1726479765002 implements MigrationInterface {
+  static entity = NftMetadataEntity;
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Based on the query it looks like it's the other way around :sweat_smile: 
```suggestion
  public async down(queryRunner: QueryRunner): Promise<void> {
```

Reply:
🤦

Reply:
05fc92cbbcb (already squashed) should solve this